pkg/activate/activator.go

```diff
@@ -21,6 +21,7 @@ import (
 type Activator interface {
 	Run(stop <-chan struct{})
 	CompareConfig(specStrategy *v1.ActivatorStrategy, specSelector labels.Selector) bool
+	GetStatus() v1.BreakerStatus
```

Comment:
the return value should be change from `v1.BreakerStatus` to `v1.ActivatorStatus` else it looks strange?

Reply:
I moved this one level above inside the Item that is collecting all breakers and activators. I have not renamed it yet, but could be done in a next step (it will change the resource definition)